src/adapters/sources/nfts.ts

```diff
@@ -23,10 +28,11 @@ export function createNFTsSource(
       return nftResults
     }
 
-    const rentalListings = await options.rentals.getOpenRentalsListingsOfNFTs(
+    const rentalListings = await options.rentals.getRentalsListingsOfNFTs(
       nftResults
         .filter((nftResult) => isLAND(nftResult))
-        .map((nftResult) => buildNftId(nftResult))
+        .map((nftResult) => buildNftId(nftResult)),
+      [RentalStatus.OPEN]
```

Comment:
Do we just want the `OPEN` ones here? Don't we need the `EXECUTED` ones as well so we get them in the `rental` object when asking for the NFTs in the frontend?

Reply:
When the source of the request comes from the graph, we enhance the NFTs with rentals. We can't really choose which rental to enhance it with using the new query parameter, as it is intended to filter NFTs by their rental status, that can only be done if the source is the signature server, so, if when asking for all NFTs we include the executed ones, the UI, that currently displays either NFTs with sales and NFT with open rents to be shown as available to sell or rent, will need to filter those who were rented and will possible break pagination.